SPECS/python-nose/python-nose.spec

```diff
@@ -0,0 +1,349 @@
+%global modname nose
+
+Summary:        Deprecated test runner for Python
+Name:           python-%{modname}
+Version:        1.3.7
+Release:        31%{?dist}
+BuildArch:      noarch
+License:        LGPLv2+ and Public Domain
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://nose.readthedocs.org/en/latest/
+Source0:        http://pypi.python.org/packages/source/n/nose/nose-%{version}.tar.gz
+# Make compatible with coverage 4.1
+# https://github.com/nose-devs/nose/pull/1004
+Patch0:         python-nose-coverage4.patch
+# Fix python 3.5 compat
+# https://github.com/nose-devs/nose/pull/983
+Patch1:         python-nose-py35.patch
+# Fix UnicodeDecodeError with captured output
+# https://github.com/nose-devs/nose/pull/988
+Patch2:         python-nose-unicode.patch
+# Allow docutils to read utf-8 source
+Patch3:         python-nose-readunicode.patch
+# Fix Python 3.6 compatibility
+# Python now returns ModuleNotFoundError instead of the previous ImportError
+# https://github.com/nose-devs/nose/pull/1029
+Patch4:         python-nose-py36.patch
+# Remove a SyntaxWarning (other projects may treat it as error)
+Patch5:         python-nose-py38.patch
+
+BuildRequires:  dos2unix
+
+%global _description %{expand:
+A deprecated test runner for Python.
+
+See https://fedoraproject.org/wiki/Changes/DeprecateNose}
+
+%description %_description
+
+%package -n python3-%{modname}
+Summary:        %{summary}
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-coverage >= 3.4-1
+Requires:       python3-setuptools
+%{?python_provide:%python_provide python3-%{modname}}
+Conflicts:      python-%{modname} < %{version}-%{release}
+Obsoletes:      python-%{modname}-docs < 1.3.7-30
+
+# This package is deprecated, no new packages in Fedora can depend on it
+# https://fedoraproject.org/wiki/Changes/DeprecateNose
+# Contact the change owners for help migrating to pytest
+Provides:       deprecated()
```

Comment:
Does this work with our build system/tdnf? I don't think I've ever seen this, and this feels like one of those weird things that might break our package graph at some point.